test/libsolidity/SemanticTest.cpp

```diff
@@ -508,17 +508,27 @@ TestCase::TestResult SemanticTest::runTest(
 			_stream << errorReporter.format(_linePrefix, _formatted);
 		}
 		_stream << endl;
+
 		AnsiColorized(_stream, _formatted, {BOLD, CYAN}) << _linePrefix << "Obtained result:" << endl;
+
 		for (TestFunctionCall const& test: m_tests)
 		{
 			ErrorReporter errorReporter;
-			_stream << test.format(
+			string outputString =
+			test.format(
 				errorReporter,
 				_linePrefix,
 				m_gasCostFailure ? TestFunctionCall::RenderMode::ExpectedValuesActualGas : TestFunctionCall::RenderMode::ActualValuesExpectedGas,
 				_formatted,
-				/* _interactivePrint */ true
-			) << endl;
+				true
+			);
+
+			auto pos = outputString.find("  // ~");
+			while ((pos=outputString.find("  // ~", pos+6)) != std::string::npos) {
+				outputString.replace(pos, 6, "\n  // ~");
+			}
```

Comment:
This feels like a hack and has potential to cause some false-positives (I don't expect `  // ~` to happen a lot in expectations but it's also not *that* unlikely and hitting this would be very confusing). Can you fix the issue at the source instead of patching the output?